app/views/editions/secondary_nav_tabs/edit/editable/components/_administration_availability.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= render "govuk_publishing_components/components/radio", {
+  heading: heading,
+  name: "edition[#{administration}_availability_attributes][authority_type]",
+  id_prefix: "conditional-checked",
```

Comment:
We should delete the `id_prefix` parameter: this is giving us multiple identical IDs and isn't used for anything. The component will ensure that we have unique IDs for all the radios.